datastore/gcloud/aio/datastore/datastore.py

```diff
@@ -336,6 +343,22 @@ async def lookup(
                 for k in data.get('deferred', [])
             ],
         }
+        if 'transaction' in data:
+            new_transaction: str = data['transaction']
+            result['transaction'] = new_transaction
+        return result
+
+    def build_read_options(self,
```

Comment:
Hmm, we should probably expose this as a `class ReadOptions` now that there are so many config options here. That said, that'd lead to a breaking change, so I think it's reasonable to implement as-is for now and improve it later. Would you mind adding a note on this in this function? eg.
```
# TODO: expose ReadOptions directly to users
# See https://cloud.google.com/datastore/docs/reference/data/rest/v1/ReadOptions
```
Thanks!

For now, please also rename this method with a leading underscore (eg. to `def _build_read_options()`) to ensure folks don't use this method directly as a public interface.